emails/models.py

```diff
@@ -49,9 +50,13 @@ def __str__(self):
 
     @staticmethod
     def subdomain_available(subdomain):
+        # valid subdomains can't start or end with a hyphen, and must be 1-63
+        # alphanumeric characters and/or hyphens
+        valid_subdomain_pattern = re.compile('^(?!-)[A-Za-z0-9-]{1,63}(?<!-)$')
```

Comment:
heh, @Vinnl mentioned the same thing. @tcinotto verified we don't have a strong preference one way or another, so since there's no *technical* reason to disallow single-character subdomains, we'll do it.

I think @tcinotto is planning to spend some time thinking of some custom terms to add to our "badwords" check - e.g., "www", or "mozilla".